.circleci/config.yml

```diff
@@ -17,8 +17,13 @@ parameters:
 jobs:
   validate:
     executor:
-      name: hmpps/java
-      tag: << pipeline.parameters.java-version >>
+      name: hmpps/java_localstack_postgres
```

Comment:
We are not using a postgres container but this is the only executor present.

It does not interfere with the build but perhaps we should move to using a postgres container as well.  Would do this in a separate PR though.

Reply:
If postgresql for integration tests, best done as a TestContainers? So started automatically for int tests.